api/buildout.cfg

```diff
@@ -62,7 +61,7 @@ site-replace = True
 [versions]
 setuptools =
 zc.buildout =
-plone.restapi = 7.0.0
+plone.restapi = 7.1.0
```

Comment:
@sneridagh you can go directly to 7.2.0 if you want. :)

Reply:
oh! :)